myapp/templates/index.html

```diff
@@ -29,6 +32,20 @@
                     text-danger
                 {% endif %}
                 ">{{ stock.change_percent|floatformat:2 }}%</td>
+                {% if request.user.is_authenticated %}
+                <td>{% if stock in profile.watchlist.all %}
+                    <form method="post" action="stock/{{ stock.symbol }}/wremove"> {% csrf_token %}
+                        <input type="hidden" name="next" value="{{ request.path }}">
+                        <button type="submit" class="btn btn-outline-danger" style="padding: 0px 6px;"><i class="fa fa-eye-slash"></i></button>
+                    </form>
+                    {% else %}
+                    <form method="post" action="stock/{{ stock.symbol }}/wadd"> {% csrf_token %}

```

Comment:
the same for URL, I'm not sure how you can change it, but It could.

Reply:
Fixed!